extra/2-piping.js

```diff
@@ -16,26 +16,38 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-function format() {
-
+function format(num) {
+  let converted = num.toString();
+  return `£${converted}`;
 }
+console.log(format(20));
+// TESTING: 
+// let numz = 20;
+// let convertedNum = numz.toString();
+// console.log(typeof (convertedNum))
 
 const startingValue = 2;
 
-// Why can this code be seen as bad practice? Comment your answer.
-let badCode =
-
-/* BETTER PRACTICE */
 
-let goodCode =
+// Why can this code be seen as bad practice? Comment your answer.
+let badCode = format(multiply(add(startingValue, 10), 2));
+console.log(badCode);
+//This could be considered bad code, because it's not very easy to ready and can get confusing very quickly!
+
+// /* BETTER PRACTICE */
+const added = add(startingValue, 10);
+const multiplied = multiply(added, 2);
+let goodCode = format(multiplied);
+console.log(goodCode);
+//This code is better, because you can see step by step what is happening, and you are also storing each value into a variable, which you could use later.
```

Comment:
Hi,
I had a look at your code and I really like your example with the good and bad code.
Its really interresting how people made the good/bad code task differntly.

Reply:
@Arbeits-Sachen I'll have to check it out(: